src/utils/Validator.js

```diff
@@ -1,42 +1,49 @@
-import MESSAGE from '../constants/Message.js';
+import DEFINITION from '../constants/Definition.js';
 
-export const Validator = {
-  isArrayLengthOver: (array, max) => {
-    if (array.length > max) {
-      throw new Error(MESSAGE.ERROR.IS_ARRAY_LENGTH_OVER);
-    }
-  },
-  isStringLengthOver: (string, max) => {
-    const arrayString = [...string];
-    if (arrayString.length > max) {
-      throw new Error(MESSAGE.ERROR.IS_STRING_LENGTH_OVER);
-    }
-  },
-  isRangeOver: (number, min, max) => {
-    if (number < min || number > max) {
-      throw new Error(MESSAGE.ERROR.IS_RANGE_OVER(min, max));
-    }
-  },
-  isDecimal: number => {
-    if (number % 1 !== 0) {
-      throw new Error(MESSAGE.ERROR.IS_DECIMAL);
-    }
-  },
-  isDuplicate: array => {
-    const set = new Set(array);
-    if (set.size !== array.length) {
-      throw new Error(MESSAGE.ERROR.IS_DUPLICATE);
-    }
+const ValidationUtils = {
+  isEmpty: string => string.trim().length === 0,
```

Comment:
사소하지만, 함수니까 이렇게 네이밍해도 좋을 것 같아요!
```suggestion
  checkEmpty: string => string.trim().length === 0,
```

Reply:
검증 로직에는 습관처럼 아무 생각없이 is~ 네이밍을 하는 경우가 많았는데, 확실히 함수의 느낌을 살려서 네이밍해도 좋을 것 같네요!
여기서는 boolean값을 반환하는 코드라서 is 네이밍을 하였는데 다음 번에는 함수 네이밍에 대해 좀 더 고민해보겠습니다😃